src/config.rs

```diff
@@ -232,13 +232,17 @@ macro_rules! make_config {
                 /// We map over the string and remove all alphanumeric, _ and - characters.
                 /// This is the fastest way (within micro-seconds) instead of using a regex (which takes mili-seconds)
                 fn _privacy_mask(value: &str) -> String {
-                    value.chars().map(|c|
+                    let mut n: u16 = 0;
+                    let mut colon_match = false;
+                    value.chars().map(|c| {
+                        n += 1;
                         match c {
-                            c if c.is_alphanumeric() => '*',
-                            '_' => '*',
-                            '-' => '*',
-                            _ => c
+                            ':' if n <= 11  => { colon_match = true ; c },
+                            '/' if n <= 13 && colon_match  => c,
```

Comment:
```suggestion
                            '/' if n <= 13 && colon_match => c,
```

removed extra space, unless this was intentional

Reply:
It looks like `cargo fmt` doesn't work inside macro's, not even for _simple_ code.
But i agree, that should be different. I have run `fmt` on that part manually, and replaced it with the right formatted code.